src/engine/docker_engine.rs

```diff
@@ -85,14 +88,16 @@ impl Engine for DockerEngine {
             ..Default::default()
         });
 
-        let config = Config {
+        let config: Config<String> = Config {
             image: Some(app.image_name.clone()),
             env: Some(
                 app.env_variables
                     .iter()
                     .map(|(k, v)| format!("{}={}", k, v))
                     .collect(),
             ),
+
+            labels: Some(build_traefik_labels(&app)),
```

Comment:
Should remove `&` from `&app` here, app is already borrowed
```suggestion
            labels: Some(build_traefik_labels(app)),
```